lollipop/types.py

```diff
@@ -413,36 +414,36 @@ def load(self, data, *args, **kwargs):
         result = []
         for idx, (item_type, item) in enumerate(zip(self.item_types, data)):
             try:
-                result.add(item_type.load(item, *args, **kwargs))
+                result.append(item_type.load(item, *args, **kwargs))
             except ValidationError as ve:
                 errors_builder.add_errors({idx: ve.messages})
         errors_builder.raise_errors()
 
         return super(Tuple, self).load(result, *args, **kwargs)
 
-    def dump(self, value, *args, **kwargs):
-        if value is MISSING or value is None:
+    def dump(self, data, *args, **kwargs):
```

Comment:
It's a common convention to call value for `dump()` method a `value` and value for `load()` method - `data`.